mtest/braille/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#=============================================================================
+#  MuseScore
+#  Music Composition & Notation
+#
+#  Copyright (C) 2012 Werner Schweer
```

Comment:
Not for new files anymore, but `2020 MusEscore BVBA`

Reply:
Addressed in b0365cb. Thank you! 👍